src/controllers/expense.controller.js

```diff
@@ -0,0 +1,84 @@
+const expenseService = require('../services/expense.service.js');
+const userService = require('../services/user.service.js');
+
+const get = (_req, res) => {
+  res.send(expenseService.getAll());
+};
+
+const getById = (req, res) => {
+  const { id } = req.params;
+
+  const expense = expenseService.getById(id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(expense);
+};
+
+const create = (req, res) => {
+  const { userId, title, amount, category, note } = req.body;
+  const existedUser = userService.getById(userId);
+
+  if (!userId || !title || !amount || !category || !note || !existedUser) {
+    res.sendStatus(400);
+  }
+
+  const newExpense = expenseService.create({
+    userId,
+    title,
+    amount,
+    category,
+    note,
+  });
+
+  res.statusCode = 201;
```

Comment:
Setting `res.statusCode` directly is not the recommended way to set the status code. Instead, use `res.status(201)` to set the status code and chain it with `send(newExpense)`.